src/meta/duplication/meta_duplication_service.cpp

```diff
@@ -308,11 +310,66 @@ void meta_duplication_service::duplication_sync(duplication_sync_rpc rpc)
     }
 }
 
-// todo(jiashuo1) wait detail implementation
 void meta_duplication_service::trigger_follower_duplicate_checkpoint(
```

Comment:
It seems that function `trigger_follower_duplicate_checkpoint` will send a create table request to follower cluster meta server. I have following question about it:
1. Is this function half-implementation? I don't figure out any trigger follower checkpoint logic in this pull request, will it be added in your further pull request? If not, I suggesting that rename this function.
2. Why create follower table in duplication process, not before the duplication process?

Reply:
1. current side(master cluster) just send trigger rpc, follower will execute some code based the rpc and duplication_option, which will in next follower side pr
2. duplication must start use a `empty` table, so force require create new table

Reply:
OK, I suggest that add a new function called `create_follower_table` or other similar name to implement sending create app request.

Reply:
This function just has `one ability`: create_follower_table, so I can rename it `create_follower_table` but add another